include/gatekeeper_net.h

```diff
@@ -297,6 +297,7 @@ struct net_config {
 
 extern uint8_t default_rss_key[GATEKEEPER_RSS_KEY_LEN];
 extern uint8_t rss_key_be[RTE_DIM(default_rss_key)];
+void randomize_rss_key(void);
```

Comment:
1. Shouldn't `randomize_rss_key()` be a `static` function?

2. `randomize_rss_key()` isn't being called anywhere.

3. Please squash this patch into the previous one since it doesn't stand on its own.